integration-tests/utils/integration-helpers.js

```diff
@@ -57,6 +57,21 @@ const createEntityInWorkspace = (page, billingProject, workspaceName, testEntity
   }, billingProject, workspaceName, testEntity)
 }
 
+const checkBucketAccess = async (page, billingProject, workspaceName, accessLevel = 'OWNER') => {
+  const details = await page.evaluate((billingProject, workspaceName) => {
+    return window.Ajax().Workspaces.workspace(billingProject, workspaceName).details()
+  }, billingProject, workspaceName)
```

Comment:
Is it necessary to pass these values through or could you do this?
```suggestion
  const details = await page.evaluate(() => {
    return window.Ajax().Workspaces.workspace(billingProject, workspaceName).details()
  })
```

Reply:
That fails with billingProject not being defined.

Reply:
passing arguments via through like this is the way we get them into scope for the function evaluation on the page; the function itself is really just a string until it's run on the page.